lib/msal-common/src/request/AuthenticationParameters.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// import { MsalAccount } from "../auth/MsalAccount";
+import { StringDict } from "../utils/MsalTypes";
+import { ClientConfigurationError } from "../error/ClientConfigurationError";
+
+/**
+ * AuthenticationParameters passed by user to retrieve a token from the server.
+ * - scopes: requested token scopes
+ * - extraScopesToConsent: additional scopes to consent 
+ * - prompt: the value of the OAuth prompt parameter
+ * - extraQueryParameters: string to string map of custom query parameters
+ * - claimsRequest: stringified claims object to request additional claims in a token
+ * - authority: authority to request tokens from
+ * - state: state parameter to ensure request/response integrity
+ * - correlationId: custom correlationId given by user
+ * - account: Account object to perform SSO
+ * - sid: session id for SSO
+ * - loginHint: login hint for SSO
+ * - forceRefresh: Forces silent requests to make network calls if true
+ */
+export type AuthenticationParameters = {
```

Comment:
is there a reason that this type has no defaults set, whereas many of the others do?

Reply:
The params are optional instead, and completely depend on desired experience, so we can't really have defaults. We should have a null check function, but that will come in a future PR.